packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/index.js

```diff
@@ -28,6 +28,13 @@ export default function SidebarNavigationScreenNavigationMenus() {
 					path: '/navigation/single',
 				} );
 			}
+			if ( name === 'core/page-list-item' && attributes.id && history ) {
+				history.push( {
+					postType: 'page',
+					postId: attributes.id,
+					path: '/navigation/single',
```

Comment:
I'm working on a PR to remove this, so we might want to check which one lands first :)